src/signals/incident-management/containers/DefaultTextsAdmin/components/DefaultTextsForm.test.tsx

```diff
@@ -24,15 +26,15 @@ describe('<DefaultTextsForm />', () => {
     render(withAppContext(<DefaultTextsForm {...props} />))
 
     expect(
-      screen.getByTestId(`defaultTextFormForm${props.index}`)
+      screen.getByTestId(`default-text-form-form${props.index}`)
```

Comment:
Should these have and `dash` before the `$`? 
Same question below.

Reply:
It's a number so I didn't it did. Still don't, but I'll change it if you think it is better.

Reply:
I'd add the dash then for consitency.

Reply:
Right, done that with all data-testids with indices in their data-testid